GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1039,6 +1039,7 @@ private void UnstageFilesClick(object sender, EventArgs e)
             if (_currentFilesList != Staged)
                 return;
             Unstage();
+            Unstaged.Focus();
```

Comment:
Seems to be unrelated to the subject of the PR.
The staged control should remain focused to allow to unstage more files.